### ISSUE TITLE
Format/style errors while reading and exporting and existing file

### ISSUE THREAD

**Issue by herna** [justkawal/excel#250]
Hi,

I just read the attached 'consumption_template.xlsx' file and export again as 'consumption_template_gen.xlsx' without any modification. The generated file contains lots of formatting errors.

[consumption_template.xlsx](https://github.com/justkawal/excel/files/12026813/consumption_template.xlsx)
[consumption_template_gen.xlsx](https://github.com/justkawal/excel/files/12026814/consumption_template_gen.xlsx)

**Comment by bakhus99** [comment]
did u found solution for this error?

**Comment by herna** [comment]
No, I simply moved to this other package in the meantime: https://pub.dev/packages/spreadsheet_decoder
It doesn't have as many options as this Excel package but at least it works good regarding the styling.
